projects/packages/boost-speed-score/src/class-speed-score.php

```diff
@@ -131,6 +141,62 @@ public function dispatch_speed_score_request( $request ) {
 		return $this->prepare_speed_score_response( $url, $score_request, $score_request_no_boost );
 	}
 
+	/**
+	 * Verify and normalize the parametesr for score grpah history request.
+	 *
+	 * @param \WP_REST_Request $request The request object.
+	 *
+	 * @return array|\WP_Error An error to return or an array with the parameters.
+	 */
+	private function process_score_graph_history_params( $request ) {
```

Comment:
If the `args` [are specified](https://developer.wordpress.org/rest-api/extending-the-rest-api/adding-custom-endpoints/#arguments) when registering the rest route, this method would not be necessary.

Reply:
I just realized the code was probably copied over from `process_url_arg`.

Reply:
Addressed in https://github.com/Automattic/jetpack/pull/32016/commits/caaf333a060641e976bd194e3f5ddd49b6025eb5